### PR TITLE
Feat/pro 4623/aktualizr info more info

### PR DESCRIPTION
**Pull request by patriotyk**

### CODE REVIEW

File: src/fsstorage.h [hunk 1]
```diff
@@ -41,8 +41,10 @@ class FSStorage : public INvStorage {
   virtual void clearEcuRegistered();
   virtual void storeInstalledVersions(const std::map<std::string, std::string>& installed_versions);
   virtual bool loadInstalledVersions(std::map<std::string, std::string>* installed_versions);
+  virtual StorageType type() { return type_; };
 
  private:
+  const StorageType type_;
```
Comment (cajun-rat) [inline src/fsstorage.h]: Could this be simplified to just:

~~~
virtual StorageType type() { return kFileSystem; };
~~~

Comment (patriotyk) [reply src/fsstorage.h]: Oh, yes You are right

File: src/aktualizr_info/main.cc [hunk 2]
```diff
@@ -32,7 +32,13 @@ int main(int argc, char **argv) {
       exit(EXIT_SUCCESS);
     }
 
-    std::string sota_config_file = vm["config"].as<std::string>();
+    std::string sota_config_file = "/usr/lib/sota/sota.toml";
+    if (vm.count("config") != 0) {
+      sota_config_file = vm["config"].as<std::string>();
+    } else if (boost::filesystem::exists("/tmp/aktualizr_config_path")) {
+      sota_config_file = Utils::readFile("/tmp/aktualizr_config_path");
```
Comment (patrickvacek) [inline src/aktualizr_info/main.cc]: What's the deal with this temp file?

Comment (patrickvacek) [reply src/aktualizr_info/main.cc]: Oh, I see, that's the way to figure out whatever config aktualizr is using. Seems alright.

**Pull request merged**